pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -40,6 +44,9 @@ type InstallOptions struct {
 	ResourceDir    string
 	PodManifestDir string
 
+	TimeoutSeconds int32
```

Comment:
time.Duration

Reply:
Don't we have an nice flag type for that?

Reply:
`(type *time.Duration) as type *int32 in argument to fs.Int32Var`

Reply:
DurationVar! didn't know that exist